roles/kustomize_deploy/tasks/install_operators.yml

```diff
@@ -166,6 +166,9 @@
           type: Ready
           status: "True"
         wait_timeout: 300
+      retries: 3
+      delay: 60
+      register: _cert_manager_operator_pods
```

Comment:
`until: _cert_manager_operator_pods is success`

Reply:
Done